<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MRemapTgz</class>
 <widget class="QWidget" name="MRemapTgz">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/tgz.png</normaloff>:/new/icons/icons/tgz.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MFileChooser" name="wdgDir" native="true"/>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeItems">
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>work</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>old path exists</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>new path exists</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblInfo">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btStart">
       <property name="text">
        <string>execute</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/work.png</normaloff>:/new/icons/icons/work.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btClose">
       <property name="text">
        <string>&amp;close</string>
       </property>
       <property name="icon">
        <iconset resource="resfile.qrc">
         <normaloff>:/new/icons/icons/stop.png</normaloff>:/new/icons/icons/stop.png</iconset>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MFileChooser</class>
   <extends>QWidget</extends>
   <header>filechooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
